### PR TITLE
UP| control copy/paste in textarea fields

### PR DESCRIPTION
**Pull request by ktalbi**
120436751

### CODE REVIEW

**Comment by romfabbro** [comment]
I think it's better to show message under the comment field than in a sweet alert

**Comment by ktalbi** [comment]
done

File: Front/app/ns_modules/ns_form/NsFormsModuleGit.js [hunk 1]
```diff
@@ -242,16 +242,27 @@ define([
...
+              var strval = $(this).val();
+              var self = this;
               if ($(this).val().length > maxlen) {
-                _this.showAlertforMaxLength(); 
-                $(this).val('');
+                 _this.showErrorForMaxLength(this);
+                 var res = strval.substring(0, 250);
+                $(this).val(res);
+                setTimeout(function(){ 
+                _this.cleantextAreaAfterError(self);
```
Comment (romfabbro) [inline Front/app/ns_modules/ns_form/NsFormsModuleGit.js]: no timeout, you have to clean error message after re edition with text length < maxlength

**Comment by ktalbi** [comment]
to test

**Comment by romfabbro** [comment]
ok

**Pull request merged**